proto/kava/evmutil/v1beta1/tx.proto

```diff
@@ -50,3 +53,28 @@ message MsgConvertERC20ToCoin {
 // MsgConvertERC20ToCoinResponse defines the response value from
 // Msg/MsgConvertERC20ToCoin.
 message MsgConvertERC20ToCoinResponse {}
+
+// MsgEVMCall encapsulates an Ethereum call as a SDK message.
+message MsgEVMCall {
+  // Hex formatted address of the recipient.
+  string to = 1;
+
+  // Abi for the contract call, used for decoding contract call data.
+  string fn_abi = 2;
+
+  // Data payload of the call in hex string.
+  string data = 3;
+
+  // amount defines the integer value of the transaction amount.
+  string amount = 4 [
+    (cosmos_proto.scalar) = "cosmos.Int",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+
+  // authority is the address of the account that must be the signer.
```

Comment:
Proposal submitted
Proposal contains a CommunityPoolProposal message
CommunityPoolProposal message contains an evm call 

So for these messages the signer should always be the community pool address? 

If it's always the same address, do we need to / why should we make the user always supply this?

If it's always the same address, are we validating at the protocol and api layer that this address is the expected one (i.e. the community module)?

Reply:
Answer to these are yes and is the intended behaviour. This method of handling government msgs aligns with the current expectations of cosmo-sdk. See https://github.com/cosmos/cosmos-sdk/blob/main/x/gov/keeper/keeper.go#L45